content/docs/tasks/telemetry/servicegraph/index.md

```diff
@@ -1,12 +1,11 @@
 ---
-title: Generating a Service Graph*
+title: Generating a Service Graph
 description: This task shows you how to generate a graph of services within an Istio mesh.
 weight: 50
 keywords: [telemetry,visualization]
 ---
 
-> \* Note: The Service Graph has been deprecated.
-> It has been replaced with [Kiali](../kiali).
+> {{< warning_icon >}} The Service Graph has been deprecated and replaced with [Kiali](../kiali).
```

Comment:
How is this (from commit https://github.com/istio/istio.io/pull/2715/commits/bdac17c2fea695d0649f2df7e58c7e93ff510102):

`The Service Graph is being deprecated in Istio 1.1 in favor of Kiali. It will be removed in the 1.2 timeframe.`